src/navigation/RootNavigator.tsx

```diff
@@ -46,28 +57,52 @@ const RootNavigator = () => {
     return 'OnboardingNavigator'
   }, [hasAccounts, l1Network])
 
+  const handleAddNew = useCallback(() => {
+    navigation.navigate('AddNewAccountNavigator')
+  }, [navigation])
+
+  const onConnectedWalletsClose = useCallback(() => {
+    dispatch(appSlice.actions.toggleConnectedWallets())
+  }, [dispatch])
+
+  const showConnectedWallets = useSelector(
+    (state: RootState) => state.app.showConnectedWallets,
+  )
+
+  useEffect(() => {
+    if (showConnectedWallets) {
+      connectedWalletsRef.current?.show()
+    }
+  }, [showConnectedWallets])
+
   return (
-    <RootStack.Navigator initialRouteName={initialRouteName}>
-      {l1Network === 'helium' ? (
-        <RootStack.Screen
-          name="HomeNavigator"
-          component={HomeNavigator}
-          options={screenOptions}
-        />
-      ) : (
+    <ConnectedWallets
```

Comment:
Little interesting having it in the navigator, maybe we should move it up one place in the tree to App? If you think it's better here we can leave it though

Reply:
Actually I feel it makes more sense to stay here since navigation is introduced in the `rootNavigator`. Going to leave for now